components/ruby/lib/chef-licensing/restful_client/cache_manager.rb

```diff
@@ -0,0 +1,48 @@
+require "active_support/cache"
+require "tmpdir" unless defined?(Dir.mktmpdir)
+require_relative "../config"
+
+module ChefLicensing
+  module RestfulClient
+    class CacheManager
+      DEFAULT_TTL = 60
```

Comment:
This could be configurable at application level

Reply:
This, you mean the DEFAULT_TTL?

Reply:
yeah